app/views/admin/_admin-orders.html.haml

```diff
@@ -0,0 +1,42 @@
+.orders-container
+  .orders-header
+    .header
+      %h3 #{@header if @header} Orders
+    .sort-by
+      .dropdown
+        %a.dropdown-toggle{ "data-toggle" => "dropdown", href: "#" }
+          Filter by Status
+          %b.caret
+        %ul.dropdown-menu
+          %li
+            = link_to 'all', admin_dashboard_path
+          - @statuses.each do |status|
+            %li
+              = link_to status, admin_dashboard_path(:status => status)
+  - @orders.each do |order|
+    .order
+      .order-info
+        %p <strong>User: </strong> #{order.user.name}
+        %p <strong>Total:</strong> #{number_to_currency(order.total_price)} 
+        %p <strong>Submitted:</strong> #{order.created_at}
+        - if order.completed? || order.cancelled?
+          %p <strong>Status:</strong> #{order.status}
```

Comment:
This looks a little similar to the code in `app/views/orders/index.hmtl.erb`. You could pull some of it out into a partial to DRY up the code.